src/org/jitsi/meet/test/MuteTest.java

```diff
@@ -157,14 +157,11 @@ public void ownerMutesParticipantAndCheck()
         action.moveToElement(elem);
         action.perform();
 
-        TestUtils.waitForDisplayedElementByXPath(
-            owner,
-            "//ul[@class='popupmenu']/li/a[@class='mutelink']",
-            5);
-
-        owner.findElement(
-                By.xpath("//ul[@class='popupmenu']/li/a[@class='mutelink']"))
-            .click();
+        TestUtils.waitMillis(1000);
```

Comment:
Whenever I see these `waitMillis(...)` I'm wondering what they're trying to fix/prevent/mitigate :) Could you please add some comments explaining why we need this? I guess it's just the `waitForDisplayedElementByXPath` that's failing or not always working, could you please document this?